internal/pkg/levee/testdata/src/example.com/tests/phi/tests.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package phi
+
+import (
+	"example.com/core"
+)
+
+func TestPhiNodeWithTaintedAndNonTaintedOperands(i *core.Innocuous) {
+	for {
+		s := core.Source{}
+		var x, y interface{} = s, i
+		if true {
+			x = y
+		}
+		// TODO: no report should be produced for i
+		core.Sink(i) // want "a source has reached a sink"
+		core.Sink(x) // want "a source has reached a sink"
```

Comment:
I'd throw in the `core.Sink(y)` that we don't want report on here, and then reduplicate the test for `y=x` phi where we *would* want `core.Sink(y)` to emit.

Reply:
> I'd throw in the `core.Sink(y)` ...

Good idea. Done.

> ... reduplicate the test for y=x phi where we would want core.Sink(y) to emit ...

Also good idea. Also done.